cnf-tests/pod-utils/oslat-runner/main.go

```diff
@@ -32,10 +32,24 @@ func main() {
 	}
 
 	mainThreadCPUs := selfCPUs.ToSlice()[0]
+	klog.Info("oslat main thread cpu: %d", mainThreadCPUs)
+
 	siblings, err := node.GetCPUSiblings(mainThreadCPUs)
 	if err != nil {
 		klog.Fatalf("failed to get main thread CPU siblings: %v", err)
 	}
+	klog.Info("oslat main thread's cpu siblings: %v", siblings)
+
+	// siblings > 0 means Hyper-threading enabled
```

Comment:
can you elaborate why?

Reply:
This is how Hyper-threading works, one physical core is treated as two logical.
two logical cores of the same physical one, are treated as siblings.
https://stackoverflow.com/questions/7274585/linux-find-out-hyper-threaded-core-id